tests/tls_handler_test.c

```diff
@@ -2672,12 +2669,23 @@ static int s_test_ecc_cert_import(struct aws_allocator *allocator, void *ctx) {
     aws_byte_buf_clean_up(&key_buf);
 
     aws_io_library_clean_up();
-#    endif /* AWS_OS_APPLE */
 
     return AWS_OP_SUCCESS;
 }
 
-AWS_TEST_CASE(test_ecc_cert_import, s_test_ecc_cert_import)
+static int s_test_ecc_p256_cert_import(struct aws_allocator *allocator, void *ctx) {
+    (void)ctx;
+    return s_test_ecc_import(allocator, "ecc-key.pem", "ecc-cert.pem");
```

Comment:
Trivial: Let's rename the cert/key pair to include 256 the way we differentiate the 384 cert/key for clarity in resources.